src/vectorchord/advanced-features/rerank-in-table.md

```diff
@@ -0,0 +1,30 @@
+# Rerank In Table
+
+To maximize performance, VectorChord stores both the quantized and original vectors in the index.
+Although the original vectors can be fetched from the table, doing so is much slower than fetching them from our highly optimized data structure.
+
+
+As a result, VectorChord's disk usage could be larger than that of alternatives. This is usually acceptable because disk space is affordable and performance is more important.
```

Comment:
It's a trick and only pgvectorscale does it as default. Most indexes always store the indexed columns.